packages/sdk-metrics/src/Instruments.ts

```diff
@@ -57,7 +57,12 @@ export class SyncInstrument {
       );
       value = Math.trunc(value);
     }
-    this._writableMetricStorage.record(value, attributes, context, hrTime());
+    this._writableMetricStorage.record(
+      value,
+      attributes,
+      context,
+      timeInputToHrTime(Date.now())
```

Comment:
It might be an overkill to call `timeInputToHrTime` for a timestamp that we definitely know it is an absolute timestamp. We may need to expose `numberToHrtime` from the core package and call with it instead.

Reply:
makes sense -- would that require a separate PR to update core, and then an update to this PR to bump the dependency version in `sdk-metrics` and start using `numberToHrtime`? or can that be done in one PR somehow?

Reply:
> would that require a separate PR to update core, and then an update to this PR to bump the dependency version in `sdk-metrics` and start using `numberToHrtime`? or can that be done in one PR somehow?

Hi, you can do that in one PR :slightly_smiling_face:
All the packages in this repo are linked via `lerna` and released together.

Reply:
fabulous! just pushed an update to export and use `numberToHrtime` 🙏

Reply:
And I see you already updated to use the already exported `millisToHrTime` 🥳